### PR TITLE
Update images-in-node-red-dashboards.md

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/blog/2023/07/images-in-node-red-dashboards.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ When designing a dashboard, images allow you to significantly enrich your conten
 
 - displaying physical tools which should be used to resolve a problem.
 
+## Easily Add Images to Node-RED Dashboards with FlowFuse’s Static Asset Service
+
+The FlowFuse static assets service offers a user-friendly and efficient way to manage your static assets within Node-RED. This service simplifies asset management by allowing users to handle everything within the same instance through a convenient interface. For detailed instructions on using the static assets service, refer to the [FlowFuse Assets Service Documentation](https://flowfuse.com/docs/user/static-asset-service/).
```
Comment (ZJvandeWeg) [inline src/blog/2023/07/images-in-node-red-dashboards.md:29]: @sumitshinde-84 Can we describe step by step how to get a logo or so on a Dashboard?

File: src/blog/2023/07/images-in-node-red-dashboards.md [hunk 2]
```diff
@@ -24,6 +24,66 @@ When designing a dashboard, images allow you to significantly enrich your conten
...
+- [node-red-contrib-os](https://flows.nodered.org/node/node-red-contrib-os) - Nodes for obtaining system information like CPU usage.
+- [node-red-contrib-string](https://flows.nodered.org/node/node-red-contrib-string) - A string manipulation node based on the lightweight stringjs library.
+- [@flowfuse/node-red-dashboard](https://flows.nodered.org/node/@flowfuse/node-red-dashboard) - A set of dashboard nodes for Node-RED.
+- [node-red-node-base64](https://flows.nodered.org/node/node-red-node-base64) - A Node-RED node to encode and decode data to and from base64.
+
+## Easily Add Images to Node-RED Dashboards with FlowFuse’s Static Asset Service
+
+[FlowFuse's static assets](https://flowforge.com/docs/user/static-asset-service/) service provides a convenient way to manage static assets, such as images, within Node-RED instances. This service streamlines asset management by allowing everything to be handled through the same interface. If you want to display images using this service, follow the steps below for a straightforward process.
+
```
Comment (ZJvandeWeg) [inline src/blog/2023/07/images-in-node-red-dashboards.md:41]: @gayanSandamal Can you improve on these instructions? It seems like it's a lot of work to use images in the dashboard?

Comment (gayanSandamal) [reply src/blog/2023/07/images-in-node-red-dashboards.md:41]: I'm looking into this

Comment (gayanSandamal) [reply src/blog/2023/07/images-in-node-red-dashboards.md]: @ZJvandeWeg Simplified instructions by grouping and reducing steps for clarity with the following [commit](https://github.com/FlowFuse/website/pull/2612/commits/2dc4502cb6a62d10d0c7bd362071cb68f5a45f80) 

### Previously
**There were 11 bullet points**
Screenshot 2024-09-26 at 18 46 11

### After
**There are only 8 bullet points by grouping simplifying all the 11 bullet points**
Screenshot 2024-09-26 at 19 17 45
Screenshot 2024-09-26 at 19 18 09

**Comment by ZJvandeWeg** [comment]
@sumitshinde-84 Does it look good for you now?

File: src/blog/2023/07/images-in-node-red-dashboards.md [hunk 3]
```diff
@@ -24,6 +24,83 @@ When designing a dashboard, images allow you to significantly enrich your conten
...
+[FlowFuse's static assets](https://flowforge.com/docs/user/static-asset-service/) service provides a simple way to manage images and other assets in Node-RED. Follow these steps to quickly add images to your Node-RED dashboard.
+
+### Steps to Add Images Using the Static Asset Service:
+
+##### 1. Access the Static Assets Service
+- Log into your FlowFuse account, navigate to your **Node-RED instance**, and click on the **Static Assets Service** tab.
+
+##### 2. Create a New Folder (Optional)
+- Click New Folder to organize your assets, provide a folder name, and **confirm**.
```
Comment (sumitshinde-84) [inline src/blog/2023/07/images-in-node-red-dashboards.md:48]:
```suggestion
- Click the **New Folder** button to create a folder that will help you organize your assets. Provide a folder name and **confirm** the creation.
```

Comment (gayanSandamal) [reply src/blog/2023/07/images-in-node-red-dashboards.md:48]: @sumitshinde-84 Your suggestion is correct. I will commit it

File: src/blog/2023/07/images-in-node-red-dashboards.md [hunk 4]
```diff
@@ -24,6 +24,83 @@ When designing a dashboard, images allow you to significantly enrich your conten
...
+- Once uploaded, click the **copy icon** next to the image to get the path for use in Node-RED.
+
+##### 5. Set Up the Image Flow in Node-RED
+- Open the Node-RED editor for the relevant instance.
+- Drag an `inject` node onto the canvas and set it to trigger immediately when the flow is deployed.
+- Drag a `read file` node, paste the copied file path in the **Filename** field, and set the output to Single buffer object.
+
+##### 6. Prepare the Image for Display
+- Add a `string` node to convert the buffer to a base64 string. Set **From** as `msg.filename` and adjust the **Method** to `getmost`.
+- Add a `change` node to configure the payload to `msg.payload`.
```
Comment (sumitshinde-84) [inline src/blog/2023/07/images-in-node-red-dashboards.md:63]:
```suggestion
- Add a `change` node and configure it to add the elements shown in the following image:
```

File: src/blog/2023/07/images-in-node-red-dashboards.md [hunk 5]
```diff
@@ -24,6 +24,83 @@ When designing a dashboard, images allow you to significantly enrich your conten
...
+
+    ![The change node showing added elements](./images/change-node.png "The change node showing added elements")
+
+##### 7. Display the Image in the Dashboard
+- Drag a `ui-template` node onto the canvas.
+- Use the following code to display the image in the dashboard header or wherever needed:
```
Comment (sumitshinde-84) [inline src/blog/2023/07/images-in-node-red-dashboards.md:69]:
```suggestion
- Add the code in `ui-template` with an `<img>` tag, configuring the `src` attribute with `msg.payload`, as shown in the following code. Alternatively, you can use the following code directly if you want to display the image in the top-left corner of your dashboard header:
```

**Comment by gayanSandamal** [comment]
> A few minor suggestions; otherwise, it looks good. Thanks, Gayan, for improving the steps.

@sumitshinde-84 Thanks for the adjustments. I've committed your suggestions

**Review by gayanSandamal — APPROVED** [review]
Looks good to me

File: src/blog/2023/07/images-in-node-red-dashboards.md [hunk 6]
```diff
@@ -24,6 +24,83 @@ When designing a dashboard, images allow you to significantly enrich your conten
...
+## Prerequisites
+
+Before proceeding, ensure that you have the following custom nodes installed, as they will be used throughout this guide:
+
+- [node-red-contrib-calc](https://flows.nodered.org/node/node-red-contrib-calc) - A Node-RED node to perform basic mathematical calculations.
+- [node-red-contrib-image-output](https://flows.nodered.org/node/node-red-contrib-image-output) - A simple way to preview and examine images in your flows.
+- [node-red-contrib-os](https://flows.nodered.org/node/node-red-contrib-os) - Nodes for obtaining system information like CPU usage.
+- [node-red-contrib-string](https://flows.nodered.org/node/node-red-contrib-string) - A string manipulation node based on the lightweight stringjs library.
+- [@flowfuse/node-red-dashboard](https://flows.nodered.org/node/@flowfuse/node-red-dashboard) - A set of dashboard nodes for Node-RED.
+- [node-red-node-base64](https://flows.nodered.org/node/node-red-node-base64) - A Node-RED node to encode and decode data to and from base64.
```
Comment (ZJvandeWeg) [inline src/blog/2023/07/images-in-node-red-dashboards.md:36]: This implies the section below needs all of these? I don't think that's true?

Comment (sumitshinde-84) [reply src/blog/2023/07/images-in-node-red-dashboards.md:36]: We only use 3 in the ff section, but the rest are used later in the article.

Comment (ZJvandeWeg) [reply src/blog/2023/07/images-in-node-red-dashboards.md:36]: Lets move it to the relevant sections

Comment (sumitshinde-84) [reply src/blog/2023/07/images-in-node-red-dashboards.md]: Done !

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**